lib/data-service.js

```diff
@@ -27,6 +27,10 @@ class DataService extends EventEmitter {
       .on('topologyDescriptionChanged', (evt) => this.emit('topologyDescriptionChanged', evt));
   }
 
+  getConnectionParams() {
+    return this.client.connectionParams;
```

Comment:
if this.client is a connection model, there are no connectionParams, only connectionOptions.